src/View3d.ts

```diff
@@ -220,8 +218,8 @@ export class View3d {
   onVolumeData(volume: Volume, channels: number[]): void {
     this.image?.updateScale();
     this.image?.onChannelLoaded(channels);
-    if (volume.isLoaded()) {
-      this.tweakpane = this.setupGui(this.canvas3d.containerdiv);
+    if (volume.isLoaded() && this.tweakpane) {
+      this.tweakpane.refresh();
```

Comment:
The consideration here was that it's possible that new channels are arriving.  The current setupGui loops over all channels to build itself.  Not sure if refresh will pull the correct data.  But I think all the Channel objects are created prior to the first onVolumeData so this is probably good!

Reply:
I think I was confusing this with the separate dat.gui of the test viewer.  The internal setupGui doesn't loop over channels! (But someday it could).  So this is even safer than I was thinking.

Reply:
Yeah - I originally added some per-channel settings, but removed them in part because they relied on complicated workarounds. (See #109). Regardless, I'm pretty sure `Channel`s are still persistent for the entire life of a `Volume`, though `MeshVolume`s and other objects I'm not thinking of might not be.